src/rtf/include/rtf/TestAssert.h

```diff
@@ -82,6 +106,29 @@
                                             RTF_SOURCEFILE(),\
                                             RTF_SOURCELINE()), dynamic_cast<RTF::TestCase*>(this)) )
 
+/** Conditional assertion with throwing failure exception.
+ * \ingroup Test Assertions
+ * \param message Message to be reported as the detail of TestMessage
+ * \param condition If this condition evaluates to \c false then the
+ *                  test failed.
+ */
+#define RTF_ASSERT_FAIL_IF(condition, message) \
+    RTF_TEST_REPORT("WARNING: RTF_ASSERT_FAIL_IF(condition, message) "\
+                    "is deprecated and will be removed. Substitue with "\
```

Comment:
Typo: "Substitue --> substitute" . Furthermore, it is always nice to be kind to strangers: I would go for "*Please* substitute". : )